pkg/utils/kubernetes/featuregates.go

```diff
@@ -39,14 +39,16 @@ var featureGateVersionRanges = map[string]*FeatureGateVersionRange{
 	"APIServerIdentity":                              {AddedInVersion: "1.20"},
 	"APIServerTracing":                               {AddedInVersion: "1.22"},
 	"AdvancedAuditing":                               {},
-	"AllowInsecureBackendProxy":                      {AddedInVersion: "1.17"},
+	"AllowInsecureBackendProxy":                      {AddedInVersion: "1.17", RemovedInVersion: "1.23"},
 	"AnyVolumeDataSource":                            {AddedInVersion: "1.18"},
 	"AppArmor":                                       {},
 	"AttachVolumeLimit":                              {RemovedInVersion: "1.21"},
 	"BalanceAttachedNodeVolumes":                     {RemovedInVersion: "1.22"},
 	"BlockVolume":                                    {RemovedInVersion: "1.21"},
-	"BoundServiceAccountTokenVolume":                 {},
+	"BoundServiceAccountTokenVolume":                 {RemovedInVersion: "1.23"},
 	"CPUManager":                                     {},
+	"CPUManagerPolicyAlphaOptions":                   {AddedInVersion: "1.22"},
```

Comment:
Shouldn't all these added features gate have `AddedInVersion=1.23`?

Reply:
🙈 Thanks for catching, sure. Fixed in ea23d09bf.